.env.template

```diff
@@ -18,67 +24,41 @@
 
 ## AUTHORISE COMMAND KEY - Key to authorise commands
 # AUTHORISE_COMMAND_KEY=y
+
 ## EXIT_KEY - Key to exit AUTO-GPT
 # EXIT_KEY=n
 
-## PLAIN_OUTPUT - Enabeling plain output will disable spinner (Default: False)
-## Note: Spinner is used to indicate that Auto-GPT is working on something in the background
+## PLAIN_OUTPUT - Plain output, which disables the spinner (Default: False)
 # PLAIN_OUTPUT=False
 
-## DISABLED_COMMAND_CATEGORIES - The list of categories of commands that are disabled. Each of the below are an option:
-## autogpt.commands.analyze_code
-## autogpt.commands.audio_text
-## autogpt.commands.execute_code
-## autogpt.commands.file_operations
-## autogpt.commands.git_operations
-## autogpt.commands.google_search
-## autogpt.commands.image_gen
-## autogpt.commands.improve_code
-## autogpt.commands.web_selenium
-## autogpt.commands.write_tests
-## autogpt.app
-## autogpt.commands.task_statuses
```

Comment:
In the user guide ([here](https://docs.agpt.co/usage/#:~:text=You%20can%20find%20the%20list%20of%20categories%20in%20your%20.env.template)):
> You can find the list of categories in your .env.template

Where can users find this list now?

Reply:
There is a note in the `options.md` that says to look in the `commands` directory.

I expect to be changing this soon.